node_frequency: 10
turn_speed: 0.3
